### ISSUE TITLE
Compiled firmware

### ISSUE THREAD

**Issue by jamine16** [vadimkozhin/st17h66-OHS-tag#2]
Would it be possible to provide a .bin file with compiled firmware so i can patch the key and flash the .bin file directly?

Thanks

**Comment by olivluca** [comment]
Look here: https://github.com/biemster/FindMy/tree/main/Lenze_ST17H66

**Comment by biemster** [comment]
yeah not to steal @vadimkozhin 's thunder, but I think that's a bit further developed in case you did not find that yet. Although I must admit it is still using the FindMy broadcast built in this repo!

**Comment by vadimkozhin** [comment]
> Would it be possible to provide a .bin file with compiled firmware so i can patch the key and flash the .bin file directly?
> 
> 
> 
> Thanks

@jamine16, Please use @biemster repo to flash chip directly whithout compiling the firmware. 

The thing is, that you'll need to pass unique generated ID to firmware. In my repo this can be done during compilation. If I provide the .bin, the ID will be burn into the firmware, and the tag will be useless.

**Comment by jamine16** [comment]
Can i not just patch the ID/Key into bin file manually like with the NRF51/52 files?


![nrf](https://github.com/vadimkozhin/st17h66-OHS-tag/assets/96849021/056c7e9b-60d0-4946-a30f-6a8d504325a2)

**Comment by jamine16** [comment]
I converted the .hex file to .bin and located the default key and replaced it with OFFLINEFINDINGPUBLICKEYHERE! as a reference point.
I will generate a new key and flash the file, hopefully it works.

![st](https://github.com/vadimkozhin/st17h66-OHS-tag/assets/96849021/7a95fd94-baf7-4f95-a2d8-059e9655b8d8)
